README.md

```diff
@@ -1 +1,7 @@
-# Design-Pattern-Assignment
\ No newline at end of file
+# Design-Pattern-Assignment
+
+1)Abstract Factory Pattern	
+
+2)Decorator Pattern	
+
+3)Strategy Pattern
```

Comment:
Decorator is already a strategy pattern. 
Where is the behavioural pattern. 
This diagram is incorrect. 
Common and Unique cannot be classes. 
How do they inherit from an interface?